store/postgres/src/relational_queries.rs

```diff
@@ -896,8 +936,7 @@ impl<'a, Conn> RunQueryDsl<Conn> for ConflictingEntityQuery<'a> {}
 #[derive(Debug, Clone, Constructor)]
 pub struct FilterQuery<'a> {
     schema: &'a str,
-    tables: Vec<&'a Table>,
-    filter: Option<EntityFilter>,
+    tables: Vec<(&'a Table, Option<QueryFilter<'a>>)>,
```

Comment:
It's a bit confusing that this field is tables and filters but is called just `tables`. Would it be too horrible to give this a longer name?

Reply:
Good point. I was just being lazy renaming things. Renamed this to `table_and_filter_pairs`